processing/src/test/java/org/apache/druid/segment/CursorFactoryProjectionTest.java

```diff
@@ -605,6 +590,10 @@ public void testProjectionSelectionTwoDims()
                     .setInterval(Intervals.ETERNITY)
                     .addDimension("a")
                     .addDimension("b")
+                    .addOrderByColumn("a", Direction.DESCENDING)
+                    .addOrderByColumn("b", Direction.DESCENDING)
+                    .setLimit(10)
+                    .setContext(Map.of(QueryContexts.QUERY_RESOURCE_ID, RESOURCE_ID))
```

Comment:
if this is required for something, it should maybe be set by `testGroupBy` or something instead of needing each test to set it specifically